### PR TITLE
docs: revised the web embeds feature page

### PR DESCRIPTION
**Pull request by bandantonio**
#### Description of Change

Revised the web embeds feature page

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Revised the web embeds feature page

### CODE REVIEW

**Review by zcbenz — CHANGES_REQUESTED** [review]
Can you make mention that we do not recommend using webview?
https://www.electronjs.org/docs/api/webview-tag#warning

File: docs/tutorial/web-embeds.md [hunk 1]
```diff
@@ -1,22 +1,50 @@
 # Web embeds in Electron
```
Comment (erickzhao) [inline docs/tutorial/web-embeds.md:1]: Do we want to keep `In Electron` here? All our are features are technically "in Electron". 😅

File: docs/tutorial/web-embeds.md [hunk 2]
```diff
@@ -1,22 +1,50 @@
...
 
-If you want to embed (third party) web content in an Electron `BrowserWindow`, there are three options available to you: `<iframe>` tags, `<webview>` tags, and `BrowserViews`. Each one offers slightly different functionality and is useful in different situations. To help you choose between these, this guide will explain the differences and capabilities of each.
+## Overview
 
-## Iframes
+If you want to embed (third party) web content in an Electron `BrowserWindow`,
+there are three options available to you: `<iframe>` tags, `<webview>` tags,
+and `BrowserViews`. Each one offers slightly different functionality and is
+useful in different situations. To help you choose between these, this guide
+explains the differences and capabilities of each option.
```
Comment (erickzhao) [inline docs/tutorial/web-embeds.md:9]:
```suggestion
If you want to embed (third-party) web content in an Electron `BrowserWindow`,
there are three options available to you: `<iframe>` tags, `<webview>` tags,
and `BrowserViews`. Each one offers slightly different functionality and is
useful in different situations. To help you choose between these, this guide
explains the differences and capabilities of each option.
```

File: docs/tutorial/web-embeds.md [hunk 3]
```diff
@@ -1,22 +1,50 @@
...
+`<webview>` element has many custom methods and events, similar to
+`webContents`, that provide you with greater control over the content.
 
-## BrowserViews
+Compared to an `<iframe>`, `<webview>` tends to be slightly slower but offers
+much greater control in loading and communicating with the third-party content
+and handling various events.
 
-[BrowserViews](../api/browser-view.md) are not part of the DOM - instead, they are created in and controlled by your main process. They are simply another layer of web content on top of your existing window. This means that they are completely separate from your own `BrowserWindow` content and that their position is not controlled by the DOM or CSS but by setting the bounds in the main process.
+## BrowserViews
```
Comment (erickzhao) [inline docs/tutorial/web-embeds.md:38]:
```suggestion
### BrowserViews
```

File: docs/tutorial/web-embeds.md [hunk 4]
```diff
@@ -1,22 +1,50 @@
...
+they are created in and controlled by your Main process. They are simply
+another layer of web content on top of your existing window. This means
+that they are completely separate from your own `BrowserWindow` content and
+their position is not controlled by the DOM or CSS. Instead, it is controlled
+by setting the bounds in the Main process.
+
+`BrowserViews` offer the greatest control over their contents, since they
+implement the `webContents` similarly to how the `BrowserWindow` does it.
+However, as `BrowserViews` are not a part of your DOM, but are overlaid on top
+of them, which means that you will have to manage their position manually.
```
Comment (erickzhao) [inline docs/tutorial/web-embeds.md:50]:
```suggestion
However, as `BrowserViews` are not a part of your DOM, but are rather overlaid on top
of them, you will have to manage their position manually.
```

**Review by erickzhao — CHANGES_REQUESTED** [review]

File: docs/tutorial/web-embeds.md [hunk 5]
```diff
@@ -35,7 +35,14 @@ Compared to an `<iframe>`, `<webview>` tends to be slightly slower but offers
 much greater control in loading and communicating with the third-party content
 and handling various events.
 
-## BrowserViews
+> Important Note:
```
Comment (erickzhao) [inline docs/tutorial/web-embeds.md:38]: The note might fit better here right above the `webview` section

**Review by erickzhao — CHANGES_REQUESTED** [review]

File: docs/tutorial/web-embeds.md [hunk 6]
```diff
@@ -1,22 +1,57 @@
...
+and only allow the capabilities you want to support.
+
+### WebViews
+
+> Important Note:
+[we do not recommend you to use use WebViews](https://www.electronjs.org/docs/api/webview-tag#warning)
+as this tag undergoes dramatic architectural changes that may affect stability
+of your application. Consider switching to alternatives, like `iframe` and
+Electron's `BrowserView` or an architecture that avoids embedded content
+by design.
```
Comment (erickzhao) [inline docs/tutorial/web-embeds.md:27]:
```suggestion
> Important Note:
[we do not recommend you to use use WebViews](https://www.electronjs.org/docs/api/webview-tag#warning),
as this tag undergoes dramatic architectural changes that may affect stability
of your application. Consider switching to alternatives, like `iframe` and
Electron's `BrowserView`, or an architecture that avoids embedded content
by design.
```

A few extra commas and we're good, I think.

**Review by erickzhao — APPROVED** [review]

**Review by erickzhao — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Pull request merged**